packages/truffle-debugger/lib/data/sagas/index.js

```diff
@@ -82,46 +88,60 @@ function *tickSaga() {
       debug("storage vars %o", storageVars);
 
       let allocation = utils.allocateDeclarations(storageVars, definitions);
+      debug("Contract definition case");
+      debug("allocation %O",allocation);
       assignments = Object.assign(
         {}, ...Object.entries(allocation.children)
           .map( ([id, storage]) => ({
-            [id]: {
-              ...(currentAssignments[id] || { ref: {} }).ref,
+            ["0:" + id]: { //using depth 0 to indicate storage
+              ...(currentAssignments["0:" + id] || { ref: {} }).ref,
```

Comment:
Part of me thinks it might be better if we made a utils function:
```javascript
function atDepth (id, depth = 0) {
  return `${depth}:${id}`;
}
```

Then we wouldn't even need the auxiliary variables like `augDeclarationId` or literal string concatenations everywhere, we could instead just call the function directly where it's used, like:

```javascript
    ...(currentAssignments[atDepth(id)] || { ref: {} }).ref,
```

Although I want a better name than `atDepth`. Thoughts?

Reply:
Good idea -- and the reverse functions (i.e. extract id, extract depth) should exist too.  As for a name... well, I'd probably call it `augment` or `augmented`, but I'll admit that might not be very clear.  Hm.  Not sure what a good name is.

Reply:
I think `augment` works for now, provided the function has a docstring. I'm into domain-specific short words, even though they require that future developers take a bit of time digesting the lingo in a particular module.

So I think if `lib/data` wants to define a new term "augment" to mean "identify some entity in the context of a given function depth", then that's totally fine. It's succinct and useful to name such a concept, given this work.

Reply:
I'm currently calling it `augmentWithDepth`, and the inverses `idFromAugmented` and `depthFromAugmented`.